src/lib.js

```diff
@@ -2902,6 +2902,12 @@ export async function getPets(profile) {
             pet.stats[stat] = (pet.stats[stat] || 0) + constants.pet_items[heldItem].stats[stat];
           }
         }
+        if ("statsPerLevel" in constants.pet_items[heldItem]) {
+          for (const stat in constants.pet_items[heldItem].statsPerLevel) {
```

Comment:
Can't you just do `for (const stat in constants.pet_items[heldItem]?.statsPerLevel) {` instead of the if statement?

Reply:
Didn't think of that, updated that and the other 2 similar IFs